src/components/BackgroundImage/BackgroundImage.tsx

```diff
@@ -15,7 +15,11 @@ const BackgroundImage = (props: WithChildren<BackgroundImageProps>) => {
     const qaAttributes = getQaAttrubutes(qa || qaIdByDefault);
 
     return (
-        <div className={b(null, className)} style={style} data-qa={qa || qaIdByDefault}>
+        <div
+            className={b(null, className)}
+            style={{...style, backgroundColor: 'transparent'}}
```

Comment:
why did you set backgroundColor here but not in css-file?

Reply:
because styles from html have higher specific than styles in css